pkg/restore/import.go

```diff
@@ -242,12 +236,12 @@ func (importer *FileImporter) Import(
 			info := regionInfo
 			// Try to download file.
 			var downloadMeta *import_sstpb.SSTMeta
-			errDownload := utils.WithRetry(importer.ctx, func() error {
+			errDownload := utils.WithRetry(cctx, func() error {
```

Comment:
should this context be `importer.ctx` here? these contexts were different

Reply:
`importer.ctx` is removed, see:
> What is more, we removed the ctx and cancel field of Importer, which is deprecated by Go here:
> 
> > Do not store Contexts inside a struct type; instead, pass a Context explicitly to each function that needs it. The Context should be the first parameter, typically named ctx.

Reply:
But the original context here won't timeout.  why choose `cctx` not `ctx` here?

Reply:
Oops...